webpack/ForemanWebhooks/Routes/Webhooks/Components/WebhookForm/Components/ForemanFormikField.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field as FormikField } from 'formik';
+import { Spinner } from '@patternfly/react-core';
+import { TypeAheadSelect } from 'patternfly-react';
+import { filter } from 'lodash';
+
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import FormField from 'foremanReact/components/common/forms/FormField';
+
+const ForemanFormikField = ({
+  name,
+  type,
+  required,
+  label,
+  labelHelp,
+  inputSizeClass,
+  rows,
+  placeholder,
+  options,
+  isLoading,
+}) => (
+  <FormikField name={name}>
+    {({
+      field, // { name, value, onChange, onBlur }
+      form: { touched, errors, setFieldValue, setFieldTouched }, // also values, setXXXX, handleXXXX, dirty, isValid, status, etc.
+    }) => {
+      const defaultSelection = (fieldValue, initialOptions) => {
```

Comment:
This is for `TypeAheadSelect` where it accepts array of object while a Formik field's value should be a simple value. Mostly for a better UX.